Govind.M/myportfolio/html.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles.css">
```

Comment:
You should add `./styles.css` because this is a relative path whereas you've given an absolute path. 
Refer this : https://blog.hubspot.com/website/html-file-path